src/libmongoc/src/mongoc/mongoc-topology-private.h

```diff
@@ -230,7 +230,3 @@ mongoc_topology_rescan_srv (mongoc_topology_t *topology);
 bool
 mongoc_topology_should_rescan_srv (mongoc_topology_t *topology);
 #endif
-
-void
-_mongoc_topology_set_server_api (mongoc_topology_t *topology,
-                                 const mongoc_server_api_t *api);
```

Comment:
> I think moving `_mongoc_topology_set_server_api` above the last `#endif` so it is inside the include guard should do the trick.

@kevinAlbs I looked at the declaration at least a dozen times and was not able to spot this. Thank you!

The declaration is now obsolete since the topology itself doesn't need to know about an Api Version at all.